File: app/src/main/java/se/asapehrsson/podtest/model/Episode.java
package se.asapehrsson.podtest.model;

public class Episode {

    private Integer id;
    private String title;
    private String description;
    private String url;
    private Program program;
    private String audiopreference;
    private String publishdateutc;
    private String imageurl;
    private String imageurltemplate;
    private Broadcast broadcast;
    private Listenpodfile listenpodfile;
    private Downloadpodfile downloadpodfile;

    public Integer getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getDescription() {
        return description;
    }

    public String getUrl() {
        return url;
    }

    public Program getProgram() {
        return program;
    }


    public String getAudiopreference() {
        return audiopreference;
    }


    public String getPublishdateutc() {
        return publishdateutc;
    }


    public String getImageurl() {
        return imageurl;
    }


    public String getImageurltemplate() {
        return imageurltemplate;
    }


    public Broadcast getBroadcast() {
        return broadcast;
    }


    public Listenpodfile getListenpodfile() {
        return listenpodfile;
    }


    public Downloadpodfile getDownloadpodfile() {
        return downloadpodfile;
    }


}

File: app/src/main/java/se/asapehrsson/podtest/data/Episode.kt
package se.asapehrsson.podtest.data

import com.google.gson.annotations.SerializedName

data class Episode(
        @SerializedName("id") val id: Int? = null,
        @SerializedName("title") val title: String? = null,
        @SerializedName("description") val description: String? = null,
        @SerializedName("url") val url: String? = null,
        @SerializedName("program") val program: Program? = null,
        @SerializedName("audiopreference") val audiopreference: String? = null,
        @SerializedName("publishdateutc") val publishdateutc: String? = null,
        @SerializedName("imageurl") val imageurl: String? = null,
        @SerializedName("imageurltemplate") val imageurltemplate: String? = null,
        @SerializedName("broadcast") val broadcast: Broadcast? = null,
        @SerializedName("listenpodfile") val listenpodfile: Listenpodfile? = null,
        @SerializedName("downloadpodfile") val downloadpodfile: Downloadpodfile? = null
)
